lib/pengine/utils.c

```diff
@@ -2337,9 +2337,76 @@ find_unfencing_devices(GListPtr candidates, GListPtr matches)
     return matches;
 }
 
+static guint
+node_priority_fencing_delay(node_t * node, pe_working_set_t * data_set)
+{
+    int member_count = 0;
+    int online_count = 0;
+    int top_priority = 0;
+    int lowest_priority = 0;
+    GListPtr gIter = NULL;
+
+    // `priority-fencing-delay` is disabled
+    if (data_set->priority_fencing_delay < 0) {
+        return -1;
```

Comment:
This will wrap to max unsigned int

Reply:
Oh, forgot to change the return type. Thanks for pointing out.